js/script.js

```diff
@@ -92,10 +92,18 @@ jqueryNoConflict(document).ready(function() {
 
   fetchActionKitData(args.actionkit, function(actionKitLayerGroups) {
     $.extend(layers, actionKitLayerGroups);
-    Object.keys(layers).forEach(function(campaignName) {
+    Object.keys(actionKitLayerGroups).forEach(function(campaignName) {
       icons[campaignName] = getEventIcon();
     });
-    initializeTabletopObject(root_spreadsheet);
+
+    fetchControlShiftData(args.controlshift, function(controlShiftLayerGroups) {
+      $.extend(layers, controlShiftLayerGroups);
+      Object.keys(controlShiftLayerGroups).forEach(function(category) {
+        icons[category] = getPetitionIcon();
+      });
+
+      initializeTabletopObject(root_spreadsheet);

```

Comment:
Yes, this is a concern of mine as well. We were reluctant to touch the code that was there before (instead settling for building on top of it), but to make everything run in parallel we will definitely have to. I'll add a story to our board.